release/llm_tests/serve/llm_eight_a10G_gpu_head_node.yaml

```diff
@@ -0,0 +1,6 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```

Comment:
can we reuse llm_4xl4 config that we already have? It is more available and cheaper I think. Also less files to maintain.